mautic/mautic
# add templates choice form_submitaction_sendemail

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | x
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

-> issue #3226 

Add new field email.send.selectemails (in red) in ```Forms > Actions > Send form results``` for style result form mail.

Selected email must contains tokens {formfield=***}.

#### Steps to test this PR:
1.  in form config create `send form results` action.
![Imgur](http://i.imgur.com/7R5mpRF.png)
2. add template mail.
![Imgur](http://i.imgur.com/jmg1NUV.png)
3. submit form `http://yourmautic.io/form/{id}`.
4. check your email ! :)
